registry/coder/modules/claude-code/README.md

```diff
@@ -26,6 +26,14 @@ module "claude-code" {
 > [!NOTE]
 > By default, this module is configured to run the embedded chat interface as a path-based application. In production, we recommend that you configure a [wildcard access URL](https://coder.com/docs/admin/setup#wildcard-access-url) and set `subdomain = true`. See [here](https://coder.com/docs/tutorials/best-practices/security-best-practices#disable-path-based-apps) for more details.
 
+<!-- TODO(major): remove this note and update the prompt configuration example in the next major release -->
```

Comment:
Do we have an issue to track these TODOs? If not, they are easily overlooked.

Reply:
I don't think so, I tried adding as much information as possible in the TODOs to avoid having this overlooked...but not sure how to enforce it 😕